src/App.js

```diff
@@ -216,10 +215,6 @@ class App extends Component {
   };
 
   handleAppStateChange = async nextAppState => {
-    if (nextAppState === 'active') {
```

Comment:
We don't want this anymore?

Reply:
Good catch. Just added it back via https://github.com/rainbow-me/rainbow/pull/2189/commits/4d8efa4a67c9618ea928f81e7642a695ced3bc02